src/routes/dashboard/dashboard.js

```diff
@@ -1,10 +1,196 @@
-import React, { Component } from 'react';
+// @flow
+import React, {
+  Component
+} from 'react';
+import {
+  Doughnut,
+  Line as LineChart
+} from 'react-chartjs-2';
+import {
+  Container,
+  Grid,
+  Segment,
+  Header,
+  Divider,
+  Table
+} from 'semantic-ui-react';
+import moment from 'moment';
+import { ProgressBar } from './progressbar';
+import './style/dashboard.scss';
 
-export default class Dashboard extends Component { // eslint-disable-line
+type Props = {};
+type State = {}
+
+export class Dashboard extends Component<Props, State> {
+  customLegend(data: Object) {
+    if (data && data.labels && data.labels.length > 0 && data.datasets.length > 0) {
+      const legend = data.labels.map((item, index) => (
+        <Table.Row key={index}>
+          <Table.Cell styleName='row'>
+            <div styleName='line-wrap'>
+              <div styleName='box' style={{ background: data.datasets[0].backgroundColor[index] }}/>
+              <div styleName='text'>{item}</div>
+            </div>
+          </Table.Cell>
+          <Table.Cell textAlign='right' styleName='row'>{data.datasets[0].data[index]}</Table.Cell>
+        </Table.Row>
+      )
+      );
+      return (
+        <Table compact singleLine styleName='table'>
+          <Table.Body>
+            {legend}
+          </Table.Body>
+        </Table>
+      );
+    }
+
+    return null;
+  }
+
+  // renderHeaderMenu() {
+  //   return (
+  //     <Fragment>
+  //       <Button color='green' floated='right'>
+  //         From
+  //       </Button>
+  //       <Button color='green' floated='right'>
+  //         To
+  //       </Button>
+  //     </Fragment>
+  //   );
+  // }
```

Comment:
remove

Reply:
ah good find. I must have missed that part.